certified-connectors/Connect2All On-Premises/apiDefinition.swagger.json

```diff
@@ -2,6 +2,23 @@
   "basePath": "/",
   "consumes": [],
   "definitions": {
+    "Actiontype": {
+      "description": "The Actiontype of the synchronous action that you wish to execute",
+      "title": "",
+      "type": "string",
+      "x-ms-dynamic-values": {
+        "operationId": "GetSynchronousActions",
+        "parameters": {
+          "Company": {
+            "parameter": "Company"
+          },
+          "Url": {
+            "parameter": "Url"
+          }
+        }
+      },
+      "x-ms-summary": "Actiontype"
```

Comment:
@GAC-AJO mentioned above. Please change it if there's no specific reason to keep it this way.

Reply:
Changed Actiontype to action type.